app.R

```diff
@@ -13,12 +13,12 @@ library(tidyverse)
 options(shiny.autoreload = TRUE)
 
 # Loading the Race results data
-race_results <- readr::read_csv("data/formula1_2021season_raceResults.csv")
-race_results <- race_results |>
-  dplyr::mutate(Track = dplyr::case_when(Track == "Netherlands" ~ "Dutch",
-                                         Track == "Brazil" ~ "Sao Paulo",
-                                         TRUE ~ Track))
+race_results <- readr::read_csv("data/formula1_2021season_raceResults.csv", col_types=c(`Fastest Lap`="character"))
 race_results$Track <- factor(race_results$Track, levels = unique(race_results$Track))
+race_results$`Fastest Lap` = paste0('00:0', race_results$`Fastest Lap`)
```

Comment:
I don't think it's necessary to parse the `Fastest Lap` column into time since we're just showing the values anyways. We can just get rid of lines 18 and 19. This solves the problem of Fastest Lap not showing up properly.

Reply:
Correct, that works